### DISCUSSION TITLE
GitHub Action  未生成 Artifacts

### DISCUSSION THREAD

**Discussion by xuanchangxian** [tw93/Pake#722] [Q&A]
![image](https://github.com/tw93/Pake/assets/10562320/644406d4-0547-41a6-8006-958d284ccb83)
![image](https://github.com/tw93/Pake/assets/10562320/0d74d930-ce9e-4cb4-a92f-e7c70e0d99d4)

**Comment by xuanchangxian** [comment]
已解决

**Reply by tw93** [reply]
一般是名称命名的问题
